src/services/users.js

```diff
@@ -0,0 +1,8 @@
+import { create } from 'apisauce'
+import { environment } from '../environments/environments'
```

Comment:
Detele it an environment variable is accesible from process object

![image](https://user-images.githubusercontent.com/26824183/66327464-7db01e80-e8f0-11e9-9b92-8b87a22bd73a.png)